features/step_definitions/admin_steps.rb

```diff
@@ -261,14 +261,21 @@ def update_settings(settings)
 
 When /^I edit known issues$/ do
   step(%{I am logged in as an admin})
-    step( %{I follow "Admin Posts"})
+    step(%{I follow "Admin Posts"})
     step(%{I follow "Known Issues" within "#main"})
     step(%{I follow "Edit"})
     step(%{I fill in "known_issue_title" with "More known problems"})
     step(%{I fill in "content" with "This is a bit of a problem, and this is too"})
     step(%{I press "Post"})
 end
 
+When /^I delete known issues$/ do
+  step(%{I am logged in as an admin})
+    step(%{I follow "Admin Posts"})
```

Comment:
okay! i'll do that now

for some reason travis is giving the `expected to find text "Actually, I meant something different"` failure again... that's the second time today